static.config.js

```diff
@@ -459,10 +498,23 @@ const buildPageAtUrl = async (
     lastPublishedAt,
   } = pageAtUrlInfo;
   if (departmentpage) {
+    let locale = 'en';
+    if (['en', 'es'].includes(client.options.headers['Accept-Language'])) {
+      locale = client.options.headers['Accept-Language'];
+    }
```

Comment:
A note, thought, or potential cleanup issue for the future... Wondering if it'd be useful to have a` coa-accepted-languages` array?

Reply:
I like the idea of getting that from Joplin instead of relying on grabbing info from our headers. Maybe a good time to clean up this kind of thing would be during https://github.com/cityofaustin/techstack/issues/3388